<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>aprssystemdialog</class>
 <widget class="QDialog" name="aprssystemdialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>502</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Edit APRS System</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="nameLabel">
       <property name="text">
        <string>Name</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="name"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="channelLabel">
       <property name="text">
        <string>Channel</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="channel"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="sourceLabel">
       <property name="text">
        <string>Source</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLineEdit" name="source">
         <property name="inputMask">
          <string/>
         </property>
         <property name="maxLength">
          <number>6</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>-</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="srcSSID">
         <property name="maximum">
          <number>15</number>
         </property>
         <property name="value">
          <number>7</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="destinationLabel">
       <property name="text">
        <string>Destination</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLineEdit" name="destination">
         <property name="text">
          <string>WIDE3</string>
         </property>
         <property name="maxLength">
          <number>6</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>-</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="destSSID">
         <property name="maximum">
          <number>15</number>
         </property>
         <property name="value">
          <number>3</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="iconLabel">
       <property name="text">
        <string>Icon</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QComboBox" name="icon"/>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="updatePeriodSLabel">
       <property name="text">
        <string>Update period [s]</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QSpinBox" name="updatePeriod">
       <property name="suffix">
        <string>s</string>
       </property>
       <property name="maximum">
        <number>6000</number>
       </property>
       <property name="singleStep">
        <number>10</number>
       </property>
       <property name="value">
        <number>300</number>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="messageLabel">
       <property name="text">
        <string>Message</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QLineEdit" name="message"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>aprssystemdialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>aprssystemdialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
